nuc_morph_analysis/analyses/density/confirm_no_corr_btn_density_and _nuclear_area.py

```diff
@@ -88,6 +56,7 @@
     ax.set_xlabel(f"{xlabel} {xunit}")
     ax.set_ylabel(f"{ylabel} {yunit}")
 
```

Comment:
Why are we using '2d_area_nuc_cell_ratio', and 'inv_cyto_density'? Which one should be our density metric?

Reply:
2d_area_nuc_cell_ratio